From 8bb4e34514c10e604134a29ab4f55acd3d4f040b Mon Sep 17 00:00:00 2001
From: William Lallemand <wlallemand@exceliance.fr>
Date: Tue, 10 Dec 2013 17:28:48 +0100
Subject: [PATCH 280/283] DOC: compression: chunk are not compressed anymore

Add the chunk issue to the list of non-working cases.
---
 doc/configuration.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/doc/configuration.txt b/doc/configuration.txt
index 712afff..1d13c1e 100644
--- a/doc/configuration.txt
+++ b/doc/configuration.txt
@@ -1991,6 +1991,8 @@ compression offload
       "Accept-Encoding" header
     * the response message is not HTTP/1.1
     * HTTP status code is not 200
+    * response header "Transfer-Encoding" contains "chunked" (Temporary
+      Workaround)
     * response contain neither a "Content-Length" header nor a
       "Transfer-Encoding" whose last value is "chunked"
     * response contains a "Content-Type" header whose first value starts with
-- 
1.8.1.5

